multirun/multirun.go

```diff
@@ -34,6 +35,7 @@ func (m multirun) unboundedExecution(ctx context.Context) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	errs := make(chan error)
+	defer close(errs)
```

Comment:
Does it really need to be closed?

Reply:
It probably doesn't need to be in this case, I prefer to have these consistent though, just in case the rest of the code gets changed in the future. If you don't think it's necessary, I'll remove it.

Reply:
Please remove it. Having it here makes me think something depends on it, when nothing does. It can be added if something will need it in the future.